src/helpers/textContent.ts

```diff
@@ -0,0 +1,20 @@
+import type { ReactTestInstance } from 'react-test-renderer';
+
+export function getTextContent(
```

Comment:
i would name the file `getTextContent`, I find it easier to find, especially for people who discover the repo